va/hci/stage4/openstackcontrolplane.yaml

```diff
@@ -75,19 +75,20 @@ spec:
       route: {}
```

Comment:
The `glance.apiOverride` needs adjusting here to match the new map.

Reply:
Done